Makefile

```diff
@@ -0,0 +1,16 @@
+
+############
+# POLICIES #
+############
+
+.PHONY: render-policies
+render-policies: ## Render policies
+	@cd render && go run . -- https://github.com/kyverno/policies/main ../content/en/policies/
```

Comment:
Needs to have the `-c` flag. Also, need to solve #929 with the `render` utility because it doesn't remove directories.

Reply:
What is the `-c` flag ?

Reply:
I added an `rm` statement to remove old content.

Reply:
`-c` flag means "clean". The `rm` command also works.